in_toto/gpg/common.py

```diff
@@ -715,7 +713,7 @@ def parse_signature_packet(data, supported_signature_types=None,
   # (see RFC4880 5.2.4.1.)
   # Below we only consider the last and favor hashed over unhashed subpackets
   # TODO: Should we warn if a we use an unhashed subpacket?
-  for subpacket_type, subpacket_data in \
+  for subpacket_type, subpacket_data, is_hashed in \
```

Comment:
This is a matter of taste, but IMHO you don't need to pass the `is_hashed` flag to `parse_subpackets` above, since you can get that info by checking which list you are taking the subpacket from, e.g.:

```python
for idx, subpacket_tuple in enumerate(unhashed_subpacket_info + hashed_subpacket_info):
  is_hashed = (idx >= len(unhashed_subpacket_info))
  subpacket_type, subpacket_data = subpacket_tuple
  ...
```

What do you think?

Reply:
Hey Lukas! Thanks for the PR review, really appreciate it! I was wondering what the reasoning behind `enumarate` and `is_hashed = (idx >= len(unhashed_subpacket_info))` is?

Also, I'm not sure if `subpacket_type, subpacket_data = subpacket_tuple` would work, as the subpacket_tuple has 3 items (type, data, is_hashed). Perhaps we could do:

```
for subpacket_tuple in (unhashed_subpacket_info + hashed_subpacket_info):
  subpacket_type, subpacket_data, is_hashed = subpacket_tuple
  ...
```
but this is pretty similar to the original. Any thoughts on how to proceed? Thank you! :)

Reply:
Sorry for not being clear. My idea was to revert your change in `util.parse_subpackets`. 

IMO there is **no** need to [pass `is_hashed`](https://github.com/in-toto/in-toto/pull/288/files#diff-d29551f739c382b7faef7df62d98c3deR248) to that function and have it added as [third element to each `parsed_subpackets` entry](https://github.com/in-toto/in-toto/pull/288/files#diff-d29551f739c382b7faef7df62d98c3deR278). 

You can get the same information by checking wether you are reading the subpacket from `unhashed_subpacket_info` or `hashed_subpacket_info` list. You can do so by using the index returned by `enumerate` [as I outlined above](https://github.com/in-toto/in-toto/pull/288#discussion_r294689806).

Reply:
Ah, I see what you mean, thanks!